suites/apis/guppyTest.js

```diff
@@ -0,0 +1,87 @@
+Feature('Guppy')
+
+const chai = require('chai');
+const expect = chai.expect;
+const apiUtil = require('../../utils/apiUtil.js');
+const guppy = require('../../services/apis/guppy/guppyService.js');
+const sheepdog = require('../../services/apis/sheepdog/sheepdogService.js');
+const nodes = require('../../utils/nodes.js');
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+Before( async (home, etl, nodes) => {
+  home.complete.login();
+});
+
+// This test fails due to what looks to me like a bug in Guppy that should be fixed. I made a ticket for it.
+xScenario('I want to make a query to Guppy, but my access token is invalid or expired.', async (I, guppy) => {
```

Comment:
```suggestion
Scenario('I want to make a query to Guppy, but my access token is invalid or expired.', async (I, guppy) => {
```

Reply:
the x means "skip". it's a codeceptjs syntax https://codecept.io/basics#skipping